conf/application.conf

```diff
@@ -281,9 +281,8 @@ swagger.api.info {
 
 voxelytics {
   staleTimeout = 20 minutes
-  elasticsearch {
+  loki {
```

Comment:
Should this appear in the migration guide or in the breaking changes section of the changelog?

Reply:
Yes. I fully forgot to add changelog and migration guide. Added now.